EBA/Taschenrechner/Inforechner/fm_Informatik.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace JustForYou_Informatik
+{
+    public partial class fm_Informatik : Form
+    {
+        public fm_Informatik()
+        {
+            InitializeComponent();
+        }
+
+        private void btn_Zahlensystem_Click(object sender, EventArgs e)
+        {
+            fm_Zahlensystem x = new fm_Zahlensystem();
+            x.ShowDialog();
+        }
+
+
+        //für das Bild
+        private void btn_Bild_Click(object sender, EventArgs e)
+        {
+            var höhe = 100; //durch Wert aus Eingabemodul ersetzen
```

Comment:
mark with a todo

Reply:
this whole class got rm again ... @JanMichels  learn 2 merge